public/translations/en.json

```diff
@@ -92,7 +92,7 @@
       "comment": {
         "label": "Please leave a comment to explain why.",
         "placeholder": "Add your thoughts...",
-        "error": "Please enter a comment"
+        "error": "Enter a comment with a maximum of 250 characters"
```

Comment:
Do we need to update the welsh version?

Reply:
Looks like I didn't push the commit 🤦‍♂️